packages/app/src/app/components/SignInModal.tsx

```diff
@@ -47,10 +49,35 @@ export const SignInModal = () => {
               bottom: 0,
               zIndex: 9999,
               background: 'rgba(0, 0, 0, 0.75)',
+
+              // OutsideClickHandler styles
+              // can't use styled(OutsideClickHandler)
+              '> div': {
+                flexGrow: 1,
+                maxWidth: '720px',
+              },
             }}
           >
             <OutsideClickHandler onOutsideClick={toggleSignInModal}>
-              <SignInModalElement redirectTo={redirectOnLogin} />
+              <Element
+                css={css({
+                  boxSizing: 'border-box',
+                  backgroundColor: '#1D1D1D',
+                  borderRadius: '4px',
+                  overflow: 'hidden',
+                  margin: 'auto',
+                  padding: '24px',
+                  width: '100%',
+
+                  // With tokens
+                  boxShadow: '2',
+                })}
+              >
+                <Stack direction="vertical" gap={64}>
+                  {/* <Element paddingBottom={6}>Welcome</Element> */}
```

Comment:
I this still needed?

Reply:
Some modals will have a header with a title, I had it here just in case. Waiting for the designs to be finished before removing it.